reboot_pending/reboot_pending.dsc.resource.json

```
@@ -16,11 +16,10 @@
   "schema": {
     "embedded": {
       "$schema": "https://json-schema.org/draft/2020-12/schema",
-      "type": "null",
+      "type": "object",
```

```suggestion
      "type": ["object", "null"],
```

In this case, we want to support both not specifying any properties during input (`null`) and the actual object.

---

Added.